modules/client-configuration/pages/autoinst-unattended.adoc

```diff
@@ -0,0 +1,62 @@
+[[autoinst-unattended]]
+= Unattended Provisioning
+
+The "bare metal" functionality allows you to register any new machine as soon as it connects to the local network, using a generic PXE boot image.
+You then go to {productname} {webui} and assign a profile to this machine.
+The next time the client boots, the operating system is installed according to that profile.
+For information about the bare metal provisioning, see xref:client-configuration:autoinst-unattended.adoc#bare-metal[Bare Metal Provisioning].
+
+If you don't want to use the bare metal functionality, you can still declare manually the system at {productname}.
+The {productname} API allows you to create profiles for systems as if they were collected by the bare metal functionality.
+For information about declaring systems with the API, see xref:client-configuration:autoinst-unattended.adoc#create-system-profile[Create a System Profile Manually].
+
+
+[[bare-metal]]
+== Bare Metal Provisioning
+
+When you have the bare metal provisioning option enabled, any bare metal client connected to the {productname} network is automatically added to the organization as soon as it is powered on.
```

Comment:
We need to check how "bare metal" is defined and understood.
This sentence is ok, when "bare metal" means a system without any OS.

If this is understood as "physical system" the sentence is wrong. This mechanism is also used with VMs

Reply:
IIRC, we'd like to get rid of the term. Unfortunately, I think it still appears in the Web UI.  Instead we'd better use "Unprovisioned System" or better(?) "Not Provisioned System".  And add "VM" or "physical" if you must differentiate.

Reply:
I am also unhappy with this functionality's name. It does not reflect what it is about. But as Karl's says, it's named like this in the UI.

"Bare metal", litterally, means physical machines, made of metal. And yes, that only would be a good reason to rename the functionality.

I will remove the second "bare metal" in that sentence, so it includes VMs.